data_processing.py

```diff
@@ -4,6 +4,44 @@
 from nltk.corpus import stopwords
 from nltk.stem import WordNetLemmatizer
 from nltk.corpus import wordnet
+import torch
+from torch.utils.data import TensorDataset, DataLoader
+import pandas as pd
+import datetime
+import itertools
+
+def format_time(elapsed):
+    '''
+    Takes a time in seconds and returns a string hh:mm:ss
+    '''
+    # Round to the nearest second.
+    elapsed_rounded = int(round((elapsed)))
+    
+    # Format as hh:mm:ss
+    return str(datetime.timedelta(seconds=elapsed_rounded))
+
+def create_dataloaders(inputs, masks, labels, batch_size):
+    input_tensor = torch.tensor(inputs, dtype=torch.int32)
+    mask_tensor = torch.tensor(masks, dtype=torch.int32)
+    labels_tensor = torch.tensor(labels, dtype=torch.float64)
+    dataset = TensorDataset(input_tensor, mask_tensor, 
+                            labels_tensor)
+    dataloader = DataLoader(dataset, batch_size=batch_size, 
+                            shuffle=True)
+    return dataloader
+
+def tokenize_function(examples, tokenizer):
+    
+    """ This function tokenizes the text in the examples dictionary.
+        We pass it to the map function of the dataset so that we can batch the tokenization for efficiency by
+        tokenizing batches in parallel.
+    """
+    return tokenizer(examples["clean_text"], padding="max_length", truncation=True)
+
+def tokenize_words(dataframe, row):
+    dataframe["tokenized_sentences"] = dataframe[row].apply(nltk.word_tokenize)
```

Comment:
If you want to access a row with an index, you should use _dataframe.iloc[]_, unless you are actually accessing a column with the name being the variable _row_, which would be confusing

Reply:
Changed argument name "row" to "column". It is less confusing now